internal/namespaces/config/commands.go

```diff
@@ -114,13 +114,6 @@ func configGetCommand() *core.Command {
 			},
 		},
 		Run: func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
```

Comment:
Switch from RawArg to a simple request 
```
type Request {
  Name string
}
```

Use positional argument and you could also add Autocomplete func